config.go

```diff
@@ -71,7 +74,10 @@ func (c *FileConfig) ConfigureFile(config *FileConfig) {
 
 // Validate returns a non-nil error if the configuration of c is invalid.
 func (c *FileConfig) Validate() error {
-	return nil
+	const baseName = "parquet.(*FileConfig)."
```

Comment:
ah my first instinct reading this was that the code was wrong because the function you're calling will always return an error. maybe `checkInvalidConfiguration` ?